lib/vector-tap/Cargo.toml

```diff
@@ -24,5 +31,8 @@ portpicker = { path = "../portpicker" }
 serde_json = { workspace = true }
 tokio = { version = "1.39.3", default-features = false, features = ["test-util"] }
 
+[features]
+api = ["dep:async-graphql"]
+
```

Comment:
nit: I think I'd like to see this at the top rather than below the dependencies, particularly since it's so small.